admin/test/features/generate_pupil_pins.feature

```diff
@@ -33,3 +33,10 @@ Feature: Generate Pupil PINs
     And I am on the generate pupil pins page
     When I click Generate PINs button
     Then I cannot see this pupil in the list of Pupil on Generate Pin list page
+
+  Scenario: Sorting Pupil list on Generate Pins page
+    Given I am logged in
+    And I navigate to generate pupil pins page
+    And I click Generate PINs button
+    When I click on the Pupil heading
+    Then I should see on the Generate Pin Pupil List page that all pupils are displayed in descending order of lastname
```

Comment:
This is a extremely long step name, there is an existing step for this already - 
'Then I should see a list of pupils sorted by surname on Generate Pins List Page'

Reply:
Scenarios again have quite a few conjunctions please try summarise steps and make the scenarios less imperative.

Reply:
Done